### PR TITLE
intro draft for certitifcate authentication

### PR DESCRIPTION
**Pull request by nickgmicrosoft**
PR API.md file: https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding?version=GBmaster&path=%2Freviews%2F4505-external-api-calls-in-authentication-user-flows%2Fapi.md&_a=preview

Adding the certificate authentication option for API connectors outbound API calls.

### CODE REVIEW

**Comment by saraswatshantanu** [comment]
Looks good to me

**Review by saraswatshantanu — APPROVED** [review]

File: api-reference/beta/resources/clientCertificateAuthentication.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: "clientCertificateAuthentication resource type"
+description: "Represents configuration for fetching a clientCertificateAuthentication in an API call."
+author: "nickgmicrosoft"
+localization_priority: Normal
+ms.prod: "microsoft-identity-platform"
```
Comment (davidmu1) [inline api-reference/beta/resources/clientCertificateAuthentication.md:6]:
```suggestion
ms.prod: "identity-and-signin"
```

File: api-reference/beta/resources/pkcs12CertificateInformation.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+title: "pkcs12CertificateInformation resource type"
+description: "Represents the public information of a Pkcs12 client certificate."
+author: "nickgmicrosoft"
+localization_priority: Normal
+ms.prod: "microsoft-identity-platform"
```
Comment (davidmu1) [inline api-reference/beta/resources/pkcs12CertificateInformation.md:6]:
```suggestion
ms.prod: "identity-and-signin"
```

File: api-reference/beta/resources/pkcs12certificate.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
+title: "pkcs12Certificate resource type"
+description: "Represents configuration for uploading a pkcs12Certificate in an API call."
+author: "nickgmicrosoft"
+localization_priority: Normal
+ms.prod: "microsoft-identity-platform"
```
Comment (davidmu1) [inline api-reference/beta/resources/pkcs12certificate.md:6]:
```suggestion
ms.prod: "identity-and-sign-in"
```

File: api-reference/beta/resources/pkcs12certificate.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+doc_type: resourcePageType
+---
+
+# pkcs12Certificate resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents configuration to **upload** a certificate when using HTTPS client-certificate authentication for calling an API connector endpoint. Client certificate authentication is a mutual certificate-based authentication, where the client provides a client certificate to an API endpoint to prove its identity. The configured certificate of an API connector is sent by Azure AD to the given API endpoint, which then validates the certificate.
```
Comment (davidmu1) [inline api-reference/beta/resources/pkcs12certificate.md:16]:
```suggestion
Represents the configuration used to **upload** a certificate when using HTTPS client-certificate authentication for calling an API connector endpoint. Client certificate authentication is a mutual certificate-based authentication, where the client provides a client certificate to an API endpoint to prove its identity. The configured certificate of an API connector is sent by Azure AD to the given API endpoint, which then validates the certificate.
```

**Review by davidmu1 — APPROVED** [review]

File: api-reference/beta/api/identityapiconnector-uploadclientcertificate.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
+---
+title: "Upload client certificate for API connector"
+description: "Upload a PKCS 12 format key (PFX) to an API connectors authentication configuration."
+localization_priority: Normal
+author: "nickgmicrosoft"
+ms.prod: "identity-and-sign-in"
+doc_type: "apiPageType"
+---
+
+# Upload client certificate for API connector
```
Comment (Lauragra) [inline api-reference/beta/api/identityapiconnector-uploadclientcertificate.md:10]: Please see our topic title guidelines. The title here should be: identityApiConnector: uploadClientCertificate.

File: api-reference/beta/resources/clientCertificateAuthentication.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
+---
+title: "clientCertificateAuthentication resource type"
```
Comment (Lauragra) [inline api-reference/beta/resources/clientCertificateAuthentication.md:2]: Please adjust the file names to be all lowercase.

File: api-reference/beta/resources/pkcs12CertificateInformation.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+The following is a JSON representation of the resource.
+<!-- {
+  "blockType": "resource",
+  "@odata.type": "microsoft.graph.pkcs12CertificateInformation"
+}
+-->
+
+``` json
+{
+    "isActive": true,
```
Comment (Lauragra) [inline api-reference/beta/resources/pkcs12CertificateInformation.md:38]: In the JSON representation, just show the type, don't include actual values.

Comment (nickgmicrosoft) [reply api-reference/beta/resources/pkcs12CertificateInformation.md:38]: Resolved

**Comment by nickgmicrosoft** [comment]
@Lauragra - addressed your latest comments. Thanks!

**Review by Lauragra — APPROVED** [review]

**Pull request merged**